kafka-rest/src/test/java/io/confluent/kafkarest/resources/v3/ProduceActionTest.java

```diff
@@ -839,7 +841,7 @@ private static ProduceAction getProduceAction(
     replay(produceControllerProvider, produceController);
 
     StreamingResponseFactory streamingResponseFactory =
-        new StreamingResponseFactory(chunkedOutputFactory, FIVE_SECONDS_MS, FIVE_SECONDS_MS);
+        new StreamingResponseFactory(chunkedOutputFactory, FIVE_SECONDS_MS, FIVE_MS, DEPTH, DEPTH);
```

Comment:
Rename these to be eg DURATION or GRACE_DURATION etc, need to go look up the values atm